GitCommands/CommitDataManager.cs

```diff
@@ -66,7 +66,7 @@ public void UpdateCommitMessage(CommitData data, string sha1, ref string error)
         {
             var module = GetModule();
             if (sha1 == null)
-                throw new ArgumentNullException("sha1");
+                throw new ArgumentNullException(nameof(sha1));
```

Comment:
:memo: Conversion of string literals to `nameof` should be sent as a single pull request with no other changes